pkg/runner/runner.go

```diff
@@ -251,8 +251,9 @@ func Run(cfg *Config) error {
 	mux.Handle("/", router)
 
 	server := http.Server{
-		Addr:    cfg.ListenAddr,
-		Handler: mux,
+		Addr:              cfg.ListenAddr,
+		Handler:           mux,
+		ReadHeaderTimeout: time.Second * 30, // To mitigate Slowloris DDoS attack. Value is arbitrary picked
```

Comment:
Will this have any effect on write/read requests that take > 30s to execute?

Reply:
Btw this is a server setting, not a client. This means that client needs to send HTTP header within 30 seconds